### DISCUSSION TITLE
Readings in prep for week 8

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#10] [Reading responses etc]
After reading/watching D'Ignazio and Klein on [Unicorns, Janitors, Ninjas, Wizards, and Rock Stars](http://ebookcentral.proquest.com/lib/pitt-ebooks/detail.action?docID=6120950), Schöch on [making data smarter and cleaner](https://journalofdigitalhumanities.org/2-3/big-smart-clean-messy-data-in-the-humanities), Rawson and Muñoz on [problems with making data "clean"](https://dhdebates.gc.cuny.edu/read/untitled-f2acf72c-a469-49d8-be35-67f9ac1e3a60/section/07154de9-4903-428e-9c61-7a92a6f22e51#ch23), and Gradek and Monk on [the power of pivot tables](https://vimeo.com/703773939), please write a short response to let us know where your mind went. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* Respond to any of the above – which you can now do directly, using threads.

Where possible, please try to point us to at least one particular passage from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by chloe-cmu** [comment]
The readings for this week had a couple of useful concepts for me. D'Ignazio and Klein mention Loukissas' conception of "data settings" rather than data sets, which I found really interesting and useful (132), because it de-normalizes the structure of datasets in general. As a person working with a big federal dataset, it's interesting to think about what I might get out of thinking of it as a data setting. D'Ignaio and Klein argue in this chapter that messy data has value and information in it and that data scientists should be trying to access that complexity. Rawson and Muñoz make a related argument: that the act of data cleaning is an act of silencing and assumes an underlying correct order. The primary corrective they offer in their project on NYPL crowdsourced menus is creating an index, responding to Anna Tsing's theory of nonscalability. Tsing argues that there are elements that are simply not scalable, often because of their complex relationship with multiple other things. Rawson and Muñoz offer an index that creates more connections, and more importantly, more flexible connections, between data entries, to expose some of the things that get hidden in the standardization of data. 

Neither D'Ignzaio or Klein, or Rawson and Muñoz, argue that this process would be an easy one. But I felt that the Schöch piece helpfully illustrated why moving away from data cleaning (which again none of the authors are calling for but just for the sake of the argument) is so hard. To be able to draw conclusions from data and to use it to understand things, it has to be standardized to a certain degree. Schöch's piece emphasizes that data has to be big (large and varied) but also smart (semi-structured, limited varieties, and clean). To be able to extract information and be able to make decisions using data requires obscuring parts of that data; finding the balance is apparently a project data scientists need to constantly return to.

**Comment by SidraNArshad** [comment]
An aspect that stuck out to me from this week’s readings was Johanna Drucker’s perspective on “capta vs data” from Schoch’s “Data in the Humanities.” The larger discussion of data’s place in humanities was engaging because of my own interpretation of data as this clinical, numbers-driven term that belongs in a realm of the humanities that I don’t really participate in (or at least participate in well). I believe that my aversion to the idea of data is captured in Drucker’s conception of “capta” as it diverges from the strictly-bounded perspective of what data is or can be. “Capta” is defined as “‘that which has been captured or gathered’” which, in my mind, removes the clearly-drawn lines of what data is. Instead of a settled, predetermined group of information that defines a set trend or summarizes a phenomenon, data can be a segment of knowledge collected to exhibit a finding—one that is open to interpret the more information that is found and later defined.

**Comment by Khushboobhutani** [comment]
A line that is going to stay with me for a long time is from Catherine and Klein’s article where they argue, “On a project that involves international development data, it means beginning not with institutional goals but with Indigenous standpoints” (15). This week’s readings hint towards both the power dynamics and “privilege hazards” involved in data projects and further illuminate varied entry points into the process of data collection. Katie Rawson raises an important concern about the usage of the term “data cleaning” itself which privileges a normative order and rejects what is different. Thinking in terms of my own project, I wonder the legitimacy of data accumulation with respect to people who are documenting violence through clandestine or other “illegitimate” ways. This leads me to think about platforms where certain kinds of data enjoy more veracity or global reach as compared to other mediums, thus opening debates about hierarchization of data that privileges one constellation of events over others, one mode of documentation over the rest. This also resonates with Schoch’s piece where he underscores the experiential aspect associated with data collection and how data collection transforms the way one interacts with data. While a heterogeneous dataset might be baffling at first, I agree with Schoch that it has the potential to generate new associations and categories especially when dealing with data in humanities.

**Comment by MustafaKandil** [comment]
The readings and video for this week talk about the effective use of data in the humanities. They put a strong emphasis on the idea that data should be accessible, scalable, and well-organized to improve our understanding of complex human phenomena. A key concept that emerged is the notion of "smart data," which involves refining and structuring data in a way that makes it meaningful analysis and interpretation. In essence, it's not just about collecting large volumes of data, but also about ensuring its relevance and applicability to the specific research questions being pursued. Data cleansing techniques were also highlighted, which involve the removal of inaccuracies and inconsistencies within datasets, ultimately improving the quality and reliability of research findings.

The readings also underscored the growing role of big data in shaping the path of humanities research. With the unprecedented volumes of data generated daily, scholars in the humanities are increasingly leveraging this vast resource to uncover patterns, trends, and insights that can provide new perspectives on human behavior, culture, and societal developments. However, the challenge lies in effectively managing this abundance of information. The readings stressed the need for robust infrastructure and tools to process and analyze big data in the humanities, enabling researchers to draw meaningful conclusions and contribute to broader scholarly discussions like from big data to smart data. 

Data in humanities is not just about having a lot of information, but also about how that information is arranged. When scholars organize their data well, it becomes easier to find and use the right information. This not only helps with research but also encourages different experts to work together and learn from each other's data. The readings showed how organizing data is important for sharing knowledge and understanding complex human things in different subjects.

**Comment by amj169** [comment]
I’m thinking about the way open data is defined in this chapter in terms of utilization and responsible practices since I’m relying on opensource data for part of my project. I think we can try to be responsible by being cognizant of what we are choosing to use and discard through transparency. I think we can do this by being transparent about the data origin, collection, and cleaning processes to be clear about the decisions that were made when it comes to who and how they were involved (tools like data diaries and data dictionaries can help achieve this). We can also accomplish this by being conscious about the ways in which we’re presenting the data through our visualizations.
